tox.ini

```diff
@@ -1,43 +1,19 @@
 [tox]
 envlist =
-    py{36,37,38}-test
+    test
     build_docs
-    codestyle
 isolated_build = true
-# This is included for testing of the template. You can remove it safely.
-skip_missing_interpreters = True
 
 [testenv]
-# Pass through the following environment variables which may be needed for the CI
-passenv = HOME, WINDIR, LC_ALL, LC_CTYPE, CC, CI, TRAVIS
-
-# Suppress display of matplotlib plots generated during docs build
-setenv = MPLBACKEND=agg
-
-# Run the tests in a temporary directory to make sure that we don't import
-# the package from the source tree
 changedir = .tmp/{envname}
-
-# tox environments are constructed with so-called 'factors' (or terms)
-# separated by hyphens, e.g. test-devdeps-cov. Lines below starting with factor:
-# will only take effect if that factor is included in the environment name. To
-# see a list of example environments that can be run, along with a description,
-# run:
-#
-#     tox -l -v
-#
-description =
-    run tests
-
 deps =
-# The following indicates which extras_require from setup.cfg will be installed
+    git+https://github.com/astrofrog/astropy@iers-package
```

Comment:
temporary?

Reply:
Yes, I've added a comment to that effect - will change this in a subsequent PR once this package is release and astropy core has been updated.